ynput/OpenPype
# Asset Usage Reporter Tool

## Changelog Description
This adds simple tool for OpenPype mode that will go over all published workfiles and print linked assets and their version:
![image](https://github.com/ynput/OpenPype/assets/33513211/decf5d5f-8848-41f8-90e0-28bf6d3f3e87)

This is created per project and can be exported in csv file or copied to clipboard in *"ASCII Human readable form"*.

## Testing notes:
1. Enable **Asset Usage Reporter**

![image](https://github.com/ynput/OpenPype/assets/33513211/a5de674c-e7be-43b8-a14a-4681684bf5c7)

2. Run it from Tray:

![image](https://github.com/ynput/OpenPype/assets/33513211/a1c34720-e02f-4751-9b3a-24f21ad88ef8)

3. Select project
4. See result
